.drone.yml

```diff
@@ -106,6 +106,7 @@ steps:
 - name: cargo-test
   image: casperlabs/node-build-u1804
   commands:
+  - make audit-as
```

Comment:
shouldn't be necessary as it is run in pre-checks

Reply:
I will change back the pipeline as per the discussion above.